src/sql/workbench/contrib/executionPlan/browser/widgets/findExpensiveNodeWidget.ts

```diff
@@ -0,0 +1,311 @@
+		operationLabel.innerText = localize('expensiveOperationLabel', 'Metric:');
+
+		this._operationNameSelectBoxContainer.appendChild(operationLabel);
+		this.container.appendChild(this._operationNameSelectBoxContainer);
+
+		this.operationNameSelectBox = new SelectBox([
+			OFF_STRING,
+			ACTUAL_ELAPSED_TIME_STRING,
+			ACTUAL_ELAPSED_CPU_TIME_STRING,
+			COST_STRING,
+			SUBTREE_COST_STRING,
+			ACTUAL_NUMBER_OF_ROWS_FOR_ALL_EXECUTIONS_STRING,
+			NUMBER_OF_ROWS_READ_STRING
+		], COST_STRING, this.contextViewService, this._operationNameSelectBoxContainer);
+
+		this.operationNameSelectBox.render(this._operationNameSelectBoxContainer);
+		this._register(attachSelectBoxStyler(this.operationNameSelectBox, this.themeService));
+
+		this._operationNameSelectBoxContainer.style.width = '200px';
+		this._operationNameSelectBoxContainer.style.marginRight = '5px';
+
+		this._register(this.operationNameSelectBox.onDidSelect(e => {
+			switch (e.selected) {
+				case ACTUAL_ELAPSED_TIME_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.ActualElapsedTime;
+					break;
+				case ACTUAL_ELAPSED_CPU_TIME_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.ActualElapsedCpuTime;
+					break;
+				case COST_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.Cost;
+					break;
+				case SUBTREE_COST_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.SubtreeCost;
+					break;
+				case ACTUAL_NUMBER_OF_ROWS_FOR_ALL_EXECUTIONS_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.ActualNumberOfRowsForAllExecutions;
+					break;
+				case NUMBER_OF_ROWS_READ_STRING:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.NumberOfRowsRead;
+					break;
+				default:
+					this._selectedExpensiveOperationType = ExpensiveOperationType.Off;
+			}
+		}));
+
+		// Apply Button
+		const findExpensiveOperationAction = new FindExpensiveOperationAction();
+		this._register(findExpensiveOperationAction);
+
+		const clearExpensiveOperationAction = new TurnOffExpensiveHighlightingOperationAction();
+		this._register(clearExpensiveOperationAction);
+
+		const cancelExpensiveOperationAction = new CancelExpensiveOperationAction();
+		this._register(cancelExpensiveOperationAction);
+
+		const self = this;
+		this.operationNameSelectBox.selectElem.onkeydown = async (ev) => {
+			if (ev.key === 'Enter') {
+				if (this._selectedExpensiveOperationType === ExpensiveOperationType.Off) {
+					await clearExpensiveOperationAction.run(self);
+				}
+				else {
+					await findExpensiveOperationAction.run(self);
+				}
+
+				this.showStoreDefaultMetricPrompt();
+			}
+			else if (ev.key === 'Escape') {
+				this.widgetController.removeWidget(self);
+			}
+		};
+
+		const applyButton = new Button(this.container, {
+			title: localize('findExpensiveOperationButtonTitle', 'Find Expensive Operation')
+		});
+		applyButton.setWidth('60px');
```

Comment:
why are we setting the width explicitly? we should let it resize automatically because the word `resize` might be longer than 60px in a different language.

Reply:
I had to set the width explicitly because the widget select box looks like this if I don't:
![image](https://user-images.githubusercontent.com/87730006/191867371-8f106b8f-5a98-44ac-937c-0b24013e8521.png)

With the width of 60px the select box looks like this:
![image](https://user-images.githubusercontent.com/87730006/191867559-de39e106-d46f-4cee-9b7b-fb927dbcb7fc.png)

Reply:
If a different language uses a longer string then the select box will look like this: 
![image](https://user-images.githubusercontent.com/87730006/191872272-a739816a-f4cb-4ddd-9533-33a99f9dff45.png)

Reply:
there must be something else that is causing the size of the button to become that wide, check with developer tools, i suspect it is the width 100% plus the flex settings. by default, the button width is just the text plus some paddings.